lib/lbt/bundle/Builder.js

```diff
@@ -19,7 +19,7 @@ const {SectionType} = require("./BundleDefinition");
 const BundleWriter = require("./BundleWriter");
 const log = require("@ui5/logger").getLogger("lbt:bundle:Builder");
 
-const copyrightCommentsPattern = /copyright|\(c\)|released under|license|\u00a9/i;
```

Comment:
What about the `AutoSplitter.js`?
It also has a "copyrightCommentsPattern" with the shorter RegExp